templates/static/js/example.js

```diff
@@ -270,6 +270,20 @@
     return scripts;
   }
 
+  function fixScroll() {
+    const titleId = window.location.hash;
+    if (titleId) {
+      const title = document.querySelector(`${titleId}`);
+      var rect = title.getBoundingClientRect();
```

Comment:
This doesn't seem needed?

Reply:
Oh no it isn't - I was trying different techniques using scrollTo or scrollIntoView. It still seems flakey though! What's the demo looking like to you?